pkg/apply/applier_test.go

```diff
@@ -689,17 +689,6 @@ var obj2 = &unstructured.Unstructured{
 	},
 }
 
```

Comment:
I would expect the events coming from the applier to change. Should we make sure this is covered in the applier tests?